docs/operation/upgrade.rst

```diff
@@ -0,0 +1,100 @@
+Upgrade Guide
+=============
+This document describes a complete and reliable upgrade path for **MetalK8s**
+and all the components that make up the stack.
+
+.. important::
+
+    - Upgrading MetalK8s is a manual process and as a result,
+      the following outlined steps must be met to the later in other to achieve
+      a successful upgrade.
+    - The output of the below-provided commands solely depends on the particular
+      MetalK8s setup you have in place.
+    - **<new_version>** and **<current_version>** are suffixes which must be
+      replaced with appropriate version number.
+
+Supported Versions
+******************
+.. note::
+
+    MetalK8 supports upgrades and downgrades **strictly** from one supported
+    minor version to another. For example;
+
+    - Upgrade from 2.0.0 to 2.0.1
+    - Upgrade from 2.0.0 to 2.1.0
```

Comment:
I think we should also add an example of downgrade (or maybe we will do it when downgrade will work)

Reply:
Yes, this makes sense.

We should add it when downgrade is ready